tests/suites/test_suite_ssl.function

```diff
@@ -339,6 +341,230 @@ int mbedtls_mock_tcp_recv_nb( void *ctx, unsigned char *buf, size_t len )
+    {
+        return ret;
+    }
+
+    if( ( ret = mbedtls_ssl_config_defaults( EP(conf),
+                    endpoint_type,
+                    MBEDTLS_SSL_TRANSPORT_STREAM,
+                    MBEDTLS_SSL_PRESET_DEFAULT ) ) != 0 )
+    {
+        return ret;
+    }
+
+    ep->cert = NULL;
+
+    return 0;
+}
+/*
+ * Initialises \p ep_cert structure and assigns it to endpoint
+ * represented by \p ep
+ *
+ * \retval  0 on success or other value on error
+ */
+int mbedtls_endpoint_certificate_init( mbedtls_endpoint *ep,
+                                       mbedtls_endpoint_certificate *ep_cert )
+{
+    int i = 0;
+    int ret = 0;
+
+    ep->cert = ep_cert;
+    mbedtls_x509_crt_init( CERT(ca_cert) );
+    mbedtls_x509_crt_init( CERT(cert) );
+    mbedtls_x509_crt_init( CERT(cert2) );
+    mbedtls_pk_init( CERT(pkey) );
+    mbedtls_pk_init( CERT(pkey2) );
+
+
+    // Load the trusted CA
+
+    for( i = 0; mbedtls_test_cas[i] != NULL; i++ )
+    {
+        if( ( ret = mbedtls_x509_crt_parse( CERT(ca_cert),
+                        ( const unsigned char* ) mbedtls_test_cas[i],
+                        mbedtls_test_cas_len[i] ) ) != 0 )
+        {
+            return ret;
+        }
+    }
+
+    for( i = 0; mbedtls_test_cas_der[i] != NULL; i++ )
+    {
+        if( ( ret = mbedtls_x509_crt_parse_der( CERT(ca_cert),
+                        ( const unsigned char* ) mbedtls_test_cas_der[i],
+                        mbedtls_test_cas_der_len[i] ) ) != 0 )
+        {
+            return ret;
+        }
+    }
+
+    // Load own certificate and private key
+
+    if( ( ret = mbedtls_x509_crt_parse( CERT(cert),
+                        ( const unsigned char* ) mbedtls_test_srv_crt_rsa,
+                        mbedtls_test_srv_crt_rsa_len ) ) != 0 )
+    {
+        return ret;
+    }
+
+    if( ( ret = mbedtls_pk_parse_key( CERT(pkey),
+                        ( const unsigned char* ) mbedtls_test_srv_key_rsa,
+                        mbedtls_test_srv_key_rsa_len, NULL, 0 ) ) != 0 )
+    {
+        return ret;
+    }
+
+    if( ( ret = mbedtls_x509_crt_parse( CERT(cert2),
+                        ( const unsigned char* ) mbedtls_test_srv_crt_ec,
+                        mbedtls_test_srv_crt_ec_len ) ) != 0 )
+    {
+        return ret;
+    }
+
+    if( ( ret = mbedtls_pk_parse_key( CERT(pkey2),
+                        ( const unsigned char* ) mbedtls_test_srv_key_ec,
+                        mbedtls_test_srv_key_ec_len, NULL, 0 ) ) != 0 )
+    {
+        return ret;
+    }
+
+
+    mbedtls_ssl_conf_ca_chain( EP(conf), CERT(ca_cert), NULL );
+
+    if( ( ret = mbedtls_ssl_conf_own_cert( EP(conf), CERT(cert), CERT(pkey) ) )
+            != 0 )
+    {
+        return ret;
+    }
+
+    if( ( ret = mbedtls_ssl_conf_own_cert( EP(conf), CERT(cert2), CERT(pkey) ) )
+            != 0 )
+    {
+        return ret;
+    }
+
+    return 0;
+}
+
+/*
+ * Deinitialises certificates from endpoint represented by \p ep
```

Comment:
Note: please don't merge this suggestion directly via github.
```suggestion
 * Deinitializes certificates from endpoint represented by \p ep
```
I believe it should be spelled otherwise.

Reply:
done